app/components/record/marc_content_advice_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Record
+  class MarcContentAdviceComponent < ViewComponent::Base
+    def initialize(document:)
+      super
+      @document = document
+      @content_advice = document.marc_field(:content_advice)
+    end
+
+    attr_reader :content_advice
+
+    def render?
+      content_advice.values.present?
+    end
+
+    def call
+      content_tag(:div, class: 'py-2 px-3 mb-2 content-advice') do
+        tag.strong('Content advice: ') +
+          content_advice.values.join("<br>")
```

Comment:
Doesn't this need to be a `safe_join` to get the HTML tag to come through as expected (or maybe something  clever is happening elsewhere)?

Reply:
I was under the impression that this is always a string. I don't believe you are supposed to put HTML in MARC

Reply:
Sorry, I'm talking about the `<br>` itself. I think it'll get rendered literally instead of as an HTML tag?

Reply:
oh. Well maybe we want to do a comma? I honestly am not sure there will ever be more than one content advice statement but I wanted to guard for it.

Reply:
Did a safe_join